charts/refarch-templates/templates/configmap.yaml

```
@@ -0,0 +1,18 @@
+{{- $dot := . }}
+{{- range $configmap := .Values.configMaps -}}
+{{ $data := dict "dot" $dot "module" $configmap }}
+{{ $checksum := $configmap.data | toYaml | sha256sum | trunc 64 }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "getFullname" $data }}
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)